new-frontiers-14/frontier-station-14
# Add FTL knockdown immunity to hostile NPCs

## About the PR



Prevents hostile NPCs from being knocked down when entering/exiting FTL.  I want what they're having.

## Why / Balance


I believe NPCs drop their weapons and generally fail to pick them up when getting knocked down on FTL.

This might be better handled with a more general "immune to X status effect" system, but for this particular problem, this seems suitable.

## How to test


1. Summon the Blood Moon (`addgamerule BluespaceBloodMoon` or something similar)
2. Warp to the blood moon, observe the NPCs as the ship enters/exits bluespace, they should not fall down.
3. Summon a Syndicate Vessel (`addgamerule BluespaceIntercepted...`)
4. Warp to the syndicate vessel, observe the NPCs as the ship enters/exits bluespace, they should not fall down.

## Media


TBD

- [  ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes


**Changelog**



:cl:
- tweak: Hostile NPCs no longer fall over on entering/exiting bluespace.
